tool/tsh/tsh.go

```diff
@@ -499,6 +499,17 @@ func Run(args []string, opts ...cliOption) error {
 	// MFA subcommands.
 	mfa := newMFACommand(app)
 
+	config := app.Command("config", "Manage application configuration")
```

Comment:
@klizhentas I don't think we reached a consensus in the ticket, this was the last comment from @r0mant: https://github.com/gravitational/teleport/issues/3734#issuecomment-866560974

Reply:
I've just pushed a change to to make it just `tsh config`.